tests/unit/UserUtilsTest.js

```diff
@@ -0,0 +1,9 @@
+import * as UserUtils from '../../src/libs/UserUtils';
+
+describe('UserUtils', () => {
+    it('should return the default avatar from the avatar url', () => {
+        const avatarURL = 'https://d2k5nsl2zxldvw.cloudfront.net/images/avatars/default-avatar_7.png';
+        const defaultAvatar = UserUtils.getDefaultAvatar(1, avatarURL);
+        expect(typeof defaultAvatar).toBe('function');
```

Comment:
> This is useless. I was expecting `accountIDHashBucket` to be returned. 

@situchan
Not sure if I understood your expectation correctly here as the function `getDefaultAvatar` does not return `accountIDHashBucket`. 
Can you please explain more about your expectation? Maybe, pointing to an existing unit test case of a similar kind can also help. Maybe I am missing something simple here.
 
> Also add some more cases including invalid ones.

When the parse fails, the function will return `undefined`. I can add a case for the invalid one.

Reply:
I was thinking the purpose of this test was to evaluate regex.
But you reused this function (getDefaultAvatar) which doesn't detect regex is right or not.
If no other way, better to remove this redundant test

Reply:
Also, since our issue here was due to lack of `lookbehind` support in older safari versions, I am wondering how a unit test case will detect this.
I am not sure if the test cases are run against specific versions of browser.
I would like to know your thoughts on this.

Reply:
Similar concern: https://github.com/Expensify/App/pull/25742#discussion_r1305707956

Reply:
Maybe create lint rule something like https://eslint.org/docs/latest/rules/no-useless-backreference but out of scope for this PR.
I am fine to merge as is for now.
cc: @Julesssss

Reply:
> Maybe create lint rule something like https://eslint.org/docs/latest/rules/no-useless-backreference but out of scope for this PR.

I think this is an interesting approach and worth considering.
But yes, I also think this is out of scope for this PR.